packages/components/src/focal-point-picker/index.js

```diff
@@ -73,13 +73,19 @@ export default function FocalPointPicker( {
 	} );
 
 	useEffect( () => {
+		const updateBounds = refUpdateBounds.current;
 		const { defaultView } = dragAreaRef.current.ownerDocument;
 		defaultView.addEventListener( 'resize', updateBounds );
-		return () => {
-			defaultView.removeEventListener( 'resize', updateBounds );
-		};
+		return () => defaultView.removeEventListener( 'resize', updateBounds );
 	}, [] );
 
+	useEffect( () => {
```

Comment:
I did do some more testing and found that could eliminate a visible reposition of the focal point. So it's done in 976b4b3. Maybe the isomorphic wrapper is extraneous because it's not easy for me to imagine this component being used in an SSR context.